app/views/logs_searches/location.html.erb

```diff
@@ -11,7 +11,7 @@
     <%= form_for @search do |f| %>
       <div class='govuk-form-group'>
         <%= f.label :term, "Select one of your organisation's locations", class: 'govuk-label' %>
-        <%= collection_select :logs_search, :term, current_organisation.locations, :id, :full_address, {}, { class: 'govuk-select' } %>
+        <%= collection_select :logs_search, :term, current_organisation.locations.order([:address]), :id, :full_address, {}, { class: 'govuk-select' } %>
```

Comment:
I would suggest putting this into the controller so the view just uses an instance variable such as `@locations`